src/handlers.js

```diff
@@ -145,8 +145,8 @@ async function jobHandler(message) {
   let sha = message.payload.details['event.head.sha'] || await context.github.repos.getShaOfCommitRef({
     owner: organization,
     repo: repository,
-    ref: `heads/${message.payload.details['event.base.repo.branch']}`,
-  });
+    ref: `refs/tags/${message.payload.details['event.version']}`,
+  }).sha;
```

Comment:
Yeah, I tried this function out separately, and it returns a giant JSON full of goodies. I think this sha gets into request url later, that's why it gets too long.